DevCycle/Models/UserConfig.swift

```diff
@@ -233,9 +233,11 @@ public struct EvalReason {
         return EvalReason(reason: "DEFAULT", details: details)
     }
 
-    public static func openFeatureEvalReason(reason: String) -> EvalReason {
-        return EvalReason(reason: reason, details: "")
+    #if DEBUG
```

Comment:
is there not `#if TEST`?

Reply:
not built in, it would require a custom implementation. Using `#if DEBUG` is recommend, since it works for all Debug builds (and is automatically set as part of the build stages in Xcode / command line)